### ISSUE TITLE
Publish GBIF, iDigBio, BioCASe observatory online

### ISSUE THREAD

**Issue by mielliott** [bio-guoda/preston#26]
We should publish a full Preston observatory (preston.acis.ufl.edu or deeplinker.bio) for GBIF, iDigBio, and BioCASe online so that it can be cited. For publishing to Zenodo, if the 50GB dataset limit becomes an issue, it's possible to request a quota increase at https://zenodo.org/support. They especially like it if a (future) paper needs to cite the data. I'm not sure about limits for the Internet Archive.

Hopefully we can publish in a similar manner to the DataONE and BHL observatories. See:
* DataOne (Zenodo) - https://zenodo.org/record/3277312#.XZZCU0ZKjD4
* DataOne (Internet Archive) - https://archive.org/details/preston-dataone
* BHL (Zenodo) - https://zenodo.org/record/3251134#.XZZBYUZKjD4
* BHL (Internet Archive) - https://archive.org/details/preston-bhl

**Comment by jhpoelen** [comment]
@mielliott had a look at this and realized that #19 would be nice to have to done before publishing the observatory. This way we don't have to go back and append the provenance logs later.

**Comment by jhpoelen** [comment]
After #19 was done, I prepared and published:

Poelen, Jorrit H. (2019). A biodiversity dataset graph: GBIF, iDigBio, BioCASe (Version 0.0.2) [Data set]. Zenodo. http://doi.org/10.5281/zenodo.3484205 with provenance hash://sha256/6387c9ebed9507a0fbba2d161e83c2da73e0d6fa6dd51fb19ac4a4ca75b839c7 .

Please note that the data files  associated with the published provenance logs/indexes can be retrieved using https://deeplinker.bio, other (future) Preston remote or using methods like rsync .

**Comment by jhpoelen** [comment]
Said observatory now available through https://archive.org/details/biodiversity-dataset-archives .
